app/styleguide/components/s-palette-list.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div :class="b()">
+    <s-palette-item
```

Comment:
- [ ] Please always use a wrapper when placing a component inside an other.

```
<div :class="b('item')">
  <s-palette-item ... />
</div>
```

This way spacing and orientation can be defined.

Reply:
Not sure if i understand, lets discuss it on tuesday.